### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-generic.git`](https://github.com/cisagov/skeleton-generic.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/domain-manager-ui.git domain-manager-ui
    cd domain-manager-ui
    git remote add skeleton https://github.com/cisagov/skeleton-generic.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add .github/workflows/build.yml .pre-commit-config.yaml 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may _append_ to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

---

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by zenine07 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,7 +4,20 @@
...
+[![Docker Pulls](https://img.shields.io/docker/pulls/cisagov/example)](https://hub.docker.com/r/cisagov/example)
+[![Docker Image Size (latest by date)](https://img.shields.io/docker/image-size/cisagov/example)](https://hub.docker.com/r/cisagov/example)
+[![Platforms](https://img.shields.io/badge/platforms-amd64%20%7C%20arm%2Fv6%20%7C%20arm%2Fv7%20%7C%20arm64%20%7C%20ppc64le%20%7C%20s390x-blue)](https://hub.docker.com/r/cisagov/skeleton-docker/tags)
+
+This is a Docker skeleton project that can be used to quickly get a
+new [cisagov](https://github.com/cisagov) GitHub Docker project
+started.  This skeleton project contains [licensing
+information](LICENSE), as well as [pre-commit hooks](https://pre-commit.com)
+and [GitHub Actions](https://github.com/features/actions) configurations
+appropriate for Docker containers and the major languages that we use.
```
Comment (dav3r) [inline README.md]: This stuff is specific to the skeleton-docker project and does not belong in here.

File: README.md [hunk 2]
```diff
@@ -4,7 +4,20 @@
 [![CodeQL](https://github.com/cisagov/domain-manager-ui/workflows/CodeQL/badge.svg)](https://github.com/cisagov/domain-manager-ui/actions/workflows/codeql-analysis.yml)
 [![Known Vulnerabilities](https://snyk.io/test/github/cisagov/domain-manager-ui/badge.svg)](https://snyk.io/test/github/cisagov/domain-manager-ui)
 
-## Setup and Build ##
+## Docker Image ##
+
+[![Docker Pulls](https://img.shields.io/docker/pulls/cisagov/example)](https://hub.docker.com/r/cisagov/example)
+[![Docker Image Size (latest by date)](https://img.shields.io/docker/image-size/cisagov/example)](https://hub.docker.com/r/cisagov/example)
+[![Platforms](https://img.shields.io/badge/platforms-amd64%20%7C%20arm%2Fv6%20%7C%20arm%2Fv7%20%7C%20arm64%20%7C%20ppc64le%20%7C%20s390x-blue)](https://hub.docker.com/r/cisagov/skeleton-docker/tags)
```
Comment (dav3r) [inline README.md]: These badges should point to the image for this project, not the skeleton "example" image.

**Review by dav3r — CHANGES_REQUESTED** [review]
In addition to the comments I noted, the README contains a bunch of skeleton info that does not apply to this project and should be updated and/or removed.

File: README.md (hunk 2 above)
Comment (zenine07) [reply README.md]: @dav3r Updated those refs

File: README.md (hunk 1 above)
Comment (zenine07) [reply README.md]: @dav3r Updated those refs

**Comment by zenine07** [comment]
@dav3r What other references are you referring to in the README? I did a look over it and didn't find anything that I haven't already updated/removed.

File: README.md (hunk 1 above)
Comment (dav3r) [reply README.md]: I still see the paragraph that begins "This is a Docker skeleton project that can..."

File: README.md (hunk 2 above)
Comment (dav3r) [reply README.md]: LGTM 👍

**Comment by dav3r** [comment]
> @dav3r What other references are you referring to in the README? I did a look over it and didn't find anything that I haven't already updated/removed.

The [section about secrets](https://github.com/cisagov/domain-manager-ui#using-secrets-with-your-container) is either not needed or not customized for this repo.

Also, all of your Docker image versions in the README should be updated to reflect the current version in `src/versions.txt`.  For an example of what I'm talking about, take a look at the version tag references in [this README for another project](https://github.com/cisagov/vdp-scanner-docker#vdp-scanner-docker-).

**Comment by zenine07** [comment]
@dav3r Thanks. Made those changes. Let me know if there's anything more I missed.

**Comment by dav3r** [comment]
- There is no explanatory text at the top of the README to explain the purpose of this repository.
- The README still has a skeleton-inherited section that begins with "This is a Docker skeleton project that can..."
- In the "Image tags" section of the README change:
  - `cisagov/domain-manager-ui:1.2.3` -> `cisagov/domain-manager-ui:1.0.0`
  - `cisagov/domain-manager-ui:1.2` -> `cisagov/domain-manager-ui:1.0`

**Comment by zenine07** [comment]
@dav3r Didn't see those other references. I have fixed those and I changed the description at the top to reflect what this project is for.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -131,6 +138,12 @@ jobs:
             echo "Initializing '$path'..."; \
             terraform init -input=false -backend=false "$path"; \
             done
+=======
+        env:
+          PACKAGE_URL: github.com/terraform-docs/terraform-docs
+          PACKAGE_VERSION: ${{ steps.setup-env.outputs.terraform-docs-version }}
+        run: go install ${PACKAGE_URL}@${PACKAGE_VERSION}
+>>>>>>> ee39ac57e39767bfb26252126c993c97c0624eec
```
Comment (dav3r) [inline .github/workflows/build.yml]: There are merge markers in here that need to be cleaned up.

File: README.md [hunk 4]
```diff
@@ -1,17 +1,29 @@
...
 [![Known Vulnerabilities](https://snyk.io/test/github/cisagov/domain-manager-ui/badge.svg)](https://snyk.io/test/github/cisagov/domain-manager-ui)
 
-## Setup and Build ##
+## Docker Image ##
+
+[![Docker Pulls](https://img.shields.io/docker/pulls/cisagov/domain-manager-ui)](https://hub.docker.com/r/cisagov/domain-manager-ui)
+[![Docker Image Size (latest by date)](https://img.shields.io/docker/image-size/cisagov/domain-manager-ui)](https://hub.docker.com/r/cisagov/domain-manager-ui)
+[![Platforms](https://img.shields.io/badge/platforms-amd64%20%7C%20arm%2Fv6%20%7C%20arm%2Fv7%20%7C%20arm64%20%7C%20ppc64le%20%7C%20s390x-blue)](https://hub.docker.com/r/cisagov/domain-manager-ui/tags)
+
+This is a docker project to interact with the
```
Comment (dav3r) [inline README.md]: Super minor, but we are trying to be good with capitalization:
```suggestion
This is a Docker project to interact with the
```

File: .github/workflows/build.yml (hunk 3 above)
Comment (zenine07) [reply .github/workflows/build.yml]: Strange, not sure how those showed up. Must have been in this commit that I didn't push https://github.com/cisagov/domain-manager-ui/pull/171/commits/7933dd8f39a0aa8c2dae268382695aaa9ba088a4. I'll go fix that.

Comment (zenine07) [reply .github/workflows/build.yml]: I have now fixed that.

File: README.md (hunk 4 above)
Comment (zenine07) [reply README.md]: I have now fixed that.

**Review by dav3r — APPROVED** [review]
LGTM now - thanks! 🚀

**Pull request merged**